src/components/React/VolumeRenderer.js

```diff
@@ -90,6 +89,12 @@ export default connect(mapStateToProps)(
                     colorMapping: this.props.colorMapping,
                     channel: this.props.channel,
                     cameraState: this.props.cameraState,
+                    path: `./assets/models/${state.selection.season}-${state.selection.tide}-${state.selection.measurement}.png`,
+                    slices: state.slices,
+                    extension: state.extension,
```

Comment:
looking at the component registration template, these weren't added - should they have been?

Reply:
Not necessarily.   What line VolumeRenderer.js:76  does is to call the update function of the Aframe component `my-loader`. All the props are stored in the **data** attribute of the Aframe component prototype. They dont need to be mapped 1:1.